test_cwl.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o pipefail
+
+red=`tput setaf 1`
+green=`tput setaf 2`
+reset=`tput sgr0`
+start() { echo travis_fold':'start:$1; echo "$green$1$reset"; }
+end() { set +v; echo travis_fold':'end:$1; echo; echo; }
+die() { set +v; echo "$red$*$reset" 1>&2 ; exit 1; }
+
+pushd .
+
+CWL_NAME=workflow.cwl
+OUTPUT_NAME=test-output-actual
+for CWL_PATH in $PWD/workflows/*/workflow.cwl; do
```

Comment:
I think Joel had a preference for cwl at the top level of the repo? Even if that's not the ideal organization from our vantage point, I'd be in favor of just accommodating that, unless you've heard this can be more flexible, or you think it's worth pushing.